search/src/main/nodejs/search_create_default_channel.js

```diff
@@ -57,8 +57,9 @@ async function createDefaultChannel(searchData) {
     try {
       console.log(`Converting uploaded image to png`);
       const pngExt = ".png";
-      const image = await Jimp.read(imageContent);
-      const imageBuffer = await image.getBufferAsync(pngMime);
+      const image = await Jimp.fromBuffer(imageContent);
+      console.log(`Created Jimp image from buffer`);
```

Comment:
remove this?

Reply:
good catch - this was another debug message to figure out where it was failing